lib/testing/test_runner.sh

```diff
@@ -251,6 +251,28 @@ get_vlan_prefix()
     (IFS=:; set -- $prefix; printf "%s:%s:%s:%x::" "$1" "$2" "$3" $(($4 + $vid * 4096)))
 }
 
+check_python()
+{
+    if [[ -z ${PYTHON:-} ]]; then
+        if command -v python3; then
+            PYTHON=python3
```

Comment:
Since you're running 'command' anyway, why not assign the output to $PYTHON?

Reply:
Couldn't it be an alias? Then it would break.

Reply:
Hmm, right, 'command' can return alias definitions as well. I don't think that's good thing, though, if an alias is defined (and actually picked up by the script) that is more likely to just lead to confusion. So maybe switch to using 'which' instead?